test/grpc/death-server.cc

```diff
@@ -73,3 +51,24 @@ int main(int argc, char** argv) {
 
   future.get();
 }
+} // namespace
+} // namespace eventuals::grpc
+
+// Should only be run from tests!
+//
+// Expects one argument.
+//
+// Expects as 'argv[1]' a string representing the file descriptor that
+// this process has inherited from its parent (the test) that can be
+// used to send the bound port of the gRPC server.
+//
+// See 'server-death-test.cc' for more details.
+int main(int argc, char** argv) {
+  // TODO(benh): use stout-flags!
+  CHECK_EQ(argc, 2) << "expecting 'pipe' to be passed as an argument";
+
+  int pipe = atoi(argv[1]);
+
+  ::eventuals::grpc::RunServer(pipe);
```

Comment:
Similar to the comment earlier, please don't put tests/functions in the `eventuals::grpc` namespace since that's not currently consistent with the rest of the codebase (only made the comment here, but same for the client death test).